### PR TITLE
838 - Accordions display child nodes twice

### LINKED ISSUE #838: Accordion children display twice

### PR DESCRIPTION
**Pull request by jaskwong**
Closes #838

### CODE REVIEW

**Review by shirleywbi — APPROVED** [review]
LGTM.

File: templates/vue/src/store/mutations.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ export function setDataset(state, dataset) {
       Object.values(value).forEach(node => {
         // Has to call this so `state.nodes` is reactive
         Vue.set(state.nodes, node.id, node)
+        node.childOrdering = [...new Set(node.childOrdering)]
```
Comment (wynnset) [inline templates/vue/src/store/mutations.js:86]: Why is this needed? 🤔

**Review by wynnset — APPROVED** [review]
Looks good to me. I'd like @narendrasss to also take a look before merging this in just to make sure.

Update: With Jasmine's latest changes, I think this is pretty straightforward now so I just merged it in.

File: templates/vue/src/store/mutations.js (hunk 1 above)
Comment (jaskwong) [reply templates/vue/src/store/mutations.js:86]: I think my intention was that if there was already a duplicate (ie. childOrdering contained the same id twice), then it stayed a duplicate even after reloading so this line was supposed to fix that for already affected nodes, I can test again maybe it's not needed

Comment (jaskwong) [reply templates/vue/src/store/mutations.js:86]: Update: I retested, turns out this line is unnecessary, that was an oversight on my part sorry about that

Comment (wynnset) [reply templates/vue/src/store/mutations.js]: No worries thanks for taking a look!
